src/app/components/board-settings-dialog/board-settings-dialog.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Inject } from '@angular/core';
+import {MatDialog, MatDialogRef, MAT_DIALOG_DATA} from '@angular/material';
+import {DataService} from '../../providers/data.service';
+import {Board} from '../navbar/objects/board.object';
+import { Subscription } from "rxjs/Subscription";
+
+@Component({
+  selector: 'app-board-settings-dialog',
+  templateUrl: './board-settings-dialog.component.html',
+  styleUrls: ['./board-settings-dialog.component.scss']
+})
+export class BoardSettingsDialogComponent {
+
+  private _selectedFile: string;
```

Comment:
I think the "selectedFile" variable was only for the windows dialog, so we probably don't need it here